module/merintr/inventry.c

```diff
@@ -671,15 +672,28 @@ void InventoryLButtonDown(HWND hwnd, BOOL fDoubleClick, int x, int y, UINT keyFl
 /************************************************************************/
 /*
  * InventoryLButtonUp:  User released left mouse button on inventory box.
+ * Call InventoryMoveCurrentItem if the mouse is within the inventory.
```

Comment:
Function comments should document the behavior, not the implementation.  The caller of this function isn't interested, and shouldn't know, what other functions this calls.  Suggest removing.

Reply:
Removed as requested.